### ISSUE TITLE
How did you figure out which handle to use?

### ISSUE THREAD

**Issue by rirze** [gabebear/yardbird#2]
First, thanks for writing a nifty app-- it's pretty complete as far as I can tell!

I'm trying to write an interface for my own library to connect to the bluetooth module. Unfortunately, I am learning about le devices at the same time as well. I don't have a well-formed idea on how to read and write to the module theoretically speaking. 

Would you mind explaining how you accomplished this in your app? I've tried to read the source code, but I'm too unfamiliar with the APIs and language itself to comprehend it. 

TLDR: Is there an easy way to figure out which handle to use for reading and writing to Mirobot's bluetooth module?

**Comment by gabebear** [comment]
I should pull out the code into a library... this project was just thrown together.

Most Bluetooth Low Engergy(BLE) devices use pretty much the same GATT interface. WLKATA is just using a standard GATT interface, so I just started scanning for devices and then turned on the robot and saw it appear. The interface they are using is standard enough that this page kinda works with the robot in Chrome. https://loginov.rocks/Web-Bluetooth-Terminal/ . I think Web-Bluetooth-Terminal doesn't work properly because of something to do with line-feed handling... but haven't dug into that yet.

I don't have a ton of time to work on this stuff, but I plan to make "Yardbird-Chrome" that lets you use Chrome on Android/MacOS/Win/ChromeBook/Linux to control the robot sometime in the next few weeks.

**Comment by rirze** [comment]
Ah that makes sense-- I'll have to look more into the standard GATT interface then. I wasn't sure which standard, if at all, they were using. 

Thanks!
